webapp/src/constants/action_types.js

```diff
@@ -1,7 +1,7 @@
 import {PLUGIN_NAME} from './manifest';
 
 export const ACTION_TYPES = {
-    OPEN_SUBSCRIPTION_MODAL: `${PLUGIN_NAME}_open_subscription_modal`,
-    CLOSE_SUBSCRIPTION_MODAL: `${PLUGIN_NAME}_close_subscription_modal`,
-    RECEIVED_SUBSCRIPTION: `${PLUGIN_NAME}_received_subscription`,
+    OPEN_SUBSCRIPTION_MODAL: PLUGIN_NAME + '_open_subscription_modal',
+    CLOSE_SUBSCRIPTION_MODAL: PLUGIN_NAME + '_close_subscription_modal',
+    RECEIVED_SUBSCRIPTION: PLUGIN_NAME + '_received_subscription',
```

Comment:
Is something in the eslint config suggesting this change?

Reply:
*Unfortunately* yes this caused eslint to fail.

Reply:
What rules caused it to fail? The config should be updated to support the better syntax

Reply:
Apparently this is a bug  https://github.com/eslint/eslint/issues/13248